### DISCUSSION TITLE
New thermistor on E3V2, how to change it in marlin?

### DISCUSSION THREAD

**Discussion by rudolfkalkan** [Jyers/Marlin#1504] [Q&A]
Hi everybody!

I just swap the hot end on E2V2, with V6 clone. Now I have a different type of thermistor, one that says * 5 : 100K thermistor - ATC Semitec 104GT-2/104NT-4-R025H42G, 4.7k pullup, #define TEMP_SENSOR_0 5.

I am on "E3V2-BLTouch-5x5-HS-v4.2.2-v1.3.5.b.bin" and it works great, but my knowledge in limited in these things, and how to change type of thermistor on firmware (and not go from scratch with new "code").

Is there some simple way?

Thank you.

**Comment by FinlessBob** [comment]
Compiling your own Jyer's firmware is easy.  Just saw this video which is the easiest one to date I have seen.
I have been compiling for a while now for exactly the things you mention about thermistor, etc hardware changes (I added a z-axis extender kit. However most vids from like teaching tech, etc made doing this a bit more complicated than I thought they needed to be once I learned how to do it.  Watch this vid and you too can do it easy.
https://www.youtube.com/watch?v=so0FfFVEmiw
